### PR TITLE
Pretty party, money, tokens, and payee in Marlowe Playground

### PR DESCRIPTION
**Pull request by nau**
Pre-submit checklist:
- Branch
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested
- If you updated any cabal files or added Haskell packages:
    - [x] `$(nix-build default.nix -A pkgsLocal.updateMaterialized)` to update the materialized Nix files
   - [x] Update `hie-*.yaml` files if needed
- If you changed any Haskell files:
   - [x] `$(nix-shell shell.nix --run fix-stylish-haskell)` to fix any formatting issues
- If you changed any Purescript files:
   - [x] `$(nix-shell shell.nix --run fix-purty)` to fix any formatting issues

Pre-merge checklist:
- [x] Someone approved it
- [x] Commits have useful messages
- [x] Review clarifications made it into the code
- [x] History is moderately tidy; or going to squash-merge

### CODE REVIEW

**Review by hrajchert — APPROVED** [review]
LGTM

**Review by palas — CHANGES_REQUESTED** [review]
Looks good. Just two style comments

File: marlowe-playground-client/src/Marlowe/Linter.purs [hunk 1]
```diff
@@ -56,6 +56,7 @@ import Monaco as Monaco
 import Simulation.Types (ContractPath, ContractPathStep(..), PrefixMap)
 import StaticAnalysis.Reachability (initialisePrefixMap, stepPrefixMap)
 import Text.Pretty (hasArgs, pretty)
+import Pretty
```
[the remark this answers is not in the document]
Comment (shmish111) [reply marlowe-playground-client/src/Marlowe/Linter.purs]: 💯

[the remark this answers is not in the document]
Comment (shmish111) [reply marlowe-playground-client/src/Marlowe/Linter.purs]: FYI we are planning on writiing these things down

**Pull request merged**